proto/Networking/Requests/RequestType.proto

```diff
@@ -181,20 +181,24 @@ enum RequestType {
 	UPDATE_PLAYER_LOCATION = 5034;              // Implemented
 	PROFANITY_FILTER_ACTION = 5035;             // ???
 	// SocialActions
-	SEARCH_PLAYER = 10000;                 // Implemented
-	SEND_FRIEND_INVITE = 10002;            // Implemented
-	CANCEL_FRIEND_INVITE = 10003;          // Implemented
-	ACCEPT_FRIEND_INVITE = 10004;          // Implemented
-	DECLINE_FRIEND_INVITE = 10005;         // Implemented
-	LIST_FRIENDS = 10006;                  // Implemented
-	LIST_OUTGOING_FRIEND_INVITES = 10007;  // Implemented
-	LIST_INCOMING_FRIEND_INVITES = 10008;  // Implemented
-	REMOVE_FRIEND = 10009;                 // Implemented
-	LIST_FRIEND_STATUS = 10010;            // Implemented
-	SEND_FACEBOOK_FRIEND_INVITE = 10011;   // Implemented
-	IS_MY_FRIEND = 10012;                  // Implemented
-	CREATE_INVITE_CODE = 10013;            // Implemented
-	GET_FACEBOOK_FRIEND_LIST = 10014;      // Implemented
-	UPDATE_FACEBOOK_STATUS = 10015;        // Implemented
-	UPDATE_NOTIFICATION = 10103;           // Implemented
+	SEARCH_PLAYER = 10000;                                     // Implemented
+	SEND_FRIEND_INVITE = 10002;                                // Implemented
+	CANCEL_FRIEND_INVITE = 10003;                              // Implemented
+	ACCEPT_FRIEND_INVITE = 10004;                              // Implemented
+	DECLINE_FRIEND_INVITE = 10005;                             // Implemented
+	LIST_FRIENDS = 10006;                                      // Implemented
+	LIST_OUTGOING_FRIEND_INVITES = 10007;                      // Implemented
+	LIST_INCOMING_FRIEND_INVITES = 10008;                      // Implemented
+	REMOVE_FRIEND = 10009;                                     // Implemented
+	LIST_FRIEND_STATUS = 10010;                                // Implemented
+	SEND_FACEBOOK_FRIEND_INVITE = 10011;                       // Implemented
+	IS_MY_FRIEND = 10012;                                      // Implemented
+	CREATE_INVITE_CODE = 10013;                                // Implemented
+	GET_FACEBOOK_FRIEND_LIST = 10014;                          // Implemented
+	UPDATE_FACEBOOK_STATUS = 10015;                            // Implemented
+	REGISTER_PUSH_NOTIFICATION_SOCIAL_ACTION = 10101;          // Implemented
+	UNREGISTER_PUSH_NOTIFICATION_SOCIAL_ACTION = 10102;        // Implemented
+	UPDATE_NOTIFICATION = 10103;                               // Implemented
+	OPT_OUT_PUSH_NOTIFICATION_CATEGORY_SOCIAL_ACTION = 10104;  // Implemented
+	GET_INBOX_SOCIAL_ACTION = 10105;                           // Implemented
```

Comment:
nope, those are not social actions, those are platform actions  
and please don't rename stuff like that.

Reply:
ok revert 👍

Reply:
https://github.com/pogosandbox/node-pogo-protos/pull/19/commits/535789e43539e79403083fef746a7292f18ffce2